From: Gabriel Valcazar <gabriel.valcazar@digi.com>
Date: Thu, 17 Oct 2019 09:45:31 +0200
Subject: [PATCH] system.conf: reduce default stop timeout to 15 seconds

We currently don't have any services that, under normal circumstances, take
over 10 seconds to stop. In exceptional cases where services hang when stopping
due to unknown circumstances, make the timeout happen earlier to avoid stalling
the entire poweroff process for 90 seconds.

https://jira.digi.com/browse/DEL-6794

Signed-off-by: Gabriel Valcazar <gabriel.valcazar@digi.com>
---
 src/core/system.conf.in | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/core/system.conf.in b/src/core/system.conf.in
index f0a59a7..c25257d 100644
--- a/src/core/system.conf.in
+++ b/src/core/system.conf.in
@@ -35,6 +35,8 @@
 #DefaultStandardError=inherit
 #DefaultTimeoutStartSec=90s
 #DefaultTimeoutStopSec=90s
+# Digi-specific
+DefaultTimeoutStopSec=15s
 #DefaultRestartSec=100ms
 #DefaultStartLimitIntervalSec=10s
 #DefaultStartLimitBurst=5
